### PR TITLE
docs: lighthouse as a service

### PR DESCRIPTION
**Pull request by asilvas**
Sponsored by GoDaddy. We use this service to monitor thousands of new or modified websites every day.

https://github.com/godaddy/lighthouse4u

### CODE REVIEW

**Comment by paulirish** [comment]
this is very cool! nice work!

I'm curious about the visualization side. Do you have anything currently setup with kibana? What else are you thinking about?

**Comment by asilvas** [comment]
Yes we do have generic visualizations broken out by groups/categories.

image

The dashboard is nothing special, but could potentially provide to help save some time.

LH4U provides test isolation (versus in cicd pipelines where slaves are often competing for resources), provides us pre-RUM synthetics enabling us to hook into our PR pipelines and block merges, and supports secured apps (basic auth, JWT, etc).

File: readme.md [hunk 1]
```diff
@@ -287,6 +287,7 @@ Other awesome open source projects that use Lighthouse.
 * **[lighthouse-security](https://github.com/voorhoede/lighthouse-security#readme)** - Run a set of security audits along with Lighthouse.
 * **[Garie](https://github.com/boyney123/garie)** — An open source tool for monitoring performance using Lighthouse,  PageSpeed Insights, [Prometheus](https://prometheus.io/), [Grafana](https://grafana.com/) and [Docker](https://www.docker.com/).
 * **[lighthouse-ci](https://github.com/andreasonny83/lighthouse-ci)** - Run Lighthouse and assert scores satisfy your custom thresholds.
+* **[lighthouse4u](https://github.com/godaddy/lighthouse4u)** - LH4U provides Google Lighthouse as a service, surfaced by both a friendly UI+API, and backed by Elastic Search for all your query and visualization needs.
```
Comment (paulirish) [inline readme.md]: "for all your query and visualization needs"
⬇️ 
"for easy querying and visualization"

**Comment by asilvas** [comment]
@paulirish It's just a basic view I setup in Kibana (trivial task), which is not currently included in LH4U. Once we have a better idea of how folks want to visualize this data we can create a Kibana dashboard for importing. Out of the box we only save to Elasticsearch; how that data is consumed/visualized is up to you.

**Comment by paulirish** [comment]
@asilvas roger that. just checking. :) thanks

**Pull request merged**